src/nvim/mbyte.c

```diff
@@ -1932,7 +1923,7 @@ bool utf_iscomposing(int c)
     {0x1082, 0x108d},
     {0x108f, 0x108f},
     {0x109a, 0x109d},
-    {0x135f, 0x135f},
+    {0x135d, 0x135f},
```

Comment:
Cannot comment on the right line:
Missing change `{0x17b6, ...}` -> `{0x17b4, ...}`